src/polkadot-test.js

```diff
@@ -0,0 +1,96 @@
+import Keyring from '@polkadot/keyring';
+import {mnemonicGenerate, cryptoWaitReady} from '@polkadot/util-crypto';
+import dock from '@docknetwork/sdk';
+import {DockService} from './services/dock';
+
+const keyring = new Keyring();
+
+// const mnemonic = mnemonicGenerate(12);
+// 'wife village evoke error day record quick donor awful pass arctic arctic';
+const mnemonic =
+  'hole dog cross program hungry blue burst raccoon differ rookie pipe auction';
+
+cryptoWaitReady().then(async () => {
+  await dock.init({
+    address: 'wss://knox-1.dock.io',
+  });
+
+  const testAccount = keyring.addFromUri(`${mnemonic}`);
+
+  window.testAccount = testAccount;
+  window.dock = dock;
+
+  dock.setAccount(testAccount);
+
+  // const data = dock.api.tx.balances.transfer('37GfhtNUJk1aJhXuGxNJsAGenteDBX3DTVAvuBZm49Kqc9wA', '10000');
+  // const fee = await data.paymentInfo(testAccount);
+
+  // window.data = fee;
+
+  console.log(fee.partialFee.toString());
+});
+
+// const pair = keyring.addFromMnemonic(mnemonic, {
```

Comment:
is all the commented code in here needed?

Reply:
I've removed this file